backend/de.metas.business/src/main/sql/postgresql/ddl/functions/getcreditlimitfordepartment.sql

```diff
@@ -0,0 +1,40 @@
+DROP FUNCTION IF EXISTS getcreditlimitfordepartment(p_m_department_id numeric,
+                                                    p_date            date)
+;
+
+CREATE OR REPLACE FUNCTION getcreditlimitfordepartment(p_m_department_id numeric,
+                                                       p_date            date) RETURNS numeric
+    STABLE
+    LANGUAGE sql
+AS
+$$
+SELECT COALESCE(SUM(lim.amount), 0)
+
+FROM C_BPartner partner
+         JOIN M_SectionCode sectionCode ON partner.m_sectioncode_id = sectionCode.m_sectioncode_id
+         JOIN m_department_sectioncode depSectionCode ON sectionCode.m_sectioncode_id = depSectionCode.m_sectioncode_id
+         JOIN M_Department dep ON depSectionCode.m_department_id = dep.m_department_id
+         JOIN C_BPartner_CreditLimit lim ON partner.c_bpartner_id = lim.c_bpartner_id
+
+
+WHERE dep.m_department_id = p_m_department_ID
+  AND ((lim.processed = 'Y' AND lim.isactive = 'Y') OR (lim.processed = 'N' AND lim.isactive = 'N'))
+  AND lim.datefrom <= p_date
+  AND NOT EXISTS(SELECT 1
+                 FROM c_bpartner_creditlimit lim2
+                 WHERE lim.c_bpartner_id = lim2.c_bpartner_id
+                   AND lim2.datefrom >= lim.datefrom
+                   AND lim2.datefrom <= p_Date
+                   AND ((lim2.processed = 'Y' AND lim2.isactive = 'Y') OR (lim2.processed = 'N' AND lim2.isactive = 'N'))
+                   AND lim2.c_bpartner_creditlimit_id != lim.c_bpartner_creditlimit_id)
+
+GROUP BY dep.m_department_id
+
+$$
+;
+
+COMMENT ON FUNCTION getcreditlimitfordepartment(numeric, date) IS 'The credit limit always comes in base currency. TEST: SELECT getcreditlimitfordepartment(2156017, now()::date);'
```

Comment:
what does "basecurrency" mean?

Reply:
currency omitted from sql, before the currency was  the accounting schema currency.

Reply:
i still don'T understand. so the amount is understand to be in the accounting schema's currency? in that case please add this into to the comment

Reply:
no, it was an info about the currency, but i remove it cause not needed in the  display.